### ISSUE TITLE
Create ZoKrates Overview

### ISSUE THREAD

**Issue by JonathanHeiss** [ZoKratesPlus/zokratesplus.github.io#11]

**Comment by lucasstuve** [comment]
@JonathanHeiss hier nochmal die sehr reduzierte Darstellung im AWS-Produkt Style: https://aws.amazon.com/de/athena/?nc2=h_ql_prod_an_ath

**Comment by JonathanHeiss** [comment]
@nikiblume @chGoodchild find the PowerPoint with figures here: https://tubcloud.tu-berlin.de/apps/onlyoffice/3531453955?filePath=%2FShared%2Fise_tubcloudise%2Fprojects%2FZoKratesPlus%2FZokPlus_Workspace%2FZoKrates_Figures.pptx

**Comment by alvaro-alonso** [comment]
@jonathanHeiss please review [the landing page](https://zokratesplus.github.io/). The only thing missing is the figure `Overview of ZokratesPlus`. Did you intend on adding it to the landing page? if so, do you have any suggestions on where? or do you have a text to add to the graph?

**Comment by JonathanHeiss** [comment]
Yes, the overview should be shown on the landing page on pole position. Visitors should see whats the project about as the first information and on one glance. Text should be taken from "about". I left a comment on #3
